cmd/openshift-install/create.go

```diff
@@ -24,39 +24,51 @@ type target struct {
 	assets  []asset.WritableAsset
 }
 
-var targets = []target{{
-	name: "Install Config",
-	command: &cobra.Command{
-		Use:   "install-config",
-		Short: "Generates the Install Config asset",
-		Long:  "",
-	},
-	assets: []asset.WritableAsset{&installconfig.InstallConfig{}},
-}, {
-	name: "Manifests",
-	command: &cobra.Command{
-		Use:   "manifests",
-		Short: "Generates the Kubernetes manifests",
-		Long:  "",
-	},
-	assets: []asset.WritableAsset{&manifests.Manifests{}, &manifests.Tectonic{}},
-}, {
-	name: "Ignition Configs",
-	command: &cobra.Command{
-		Use:   "ignition-configs",
-		Short: "Generates the Ignition Config asset",
-		Long:  "",
-	},
-	assets: []asset.WritableAsset{&bootstrap.Bootstrap{}, &machine.Master{}, &machine.Worker{}},
-}, {
-	name: "Cluster",
-	command: &cobra.Command{
-		Use:   "cluster",
-		Short: "Create an OpenShift cluster",
-		Long:  "",
-	},
-	assets: []asset.WritableAsset{&cluster.TerraformVariables{}, &kubeconfig.Admin{}, &cluster.Cluster{}},
-}}
+// each target is a variable to preserve the order when creating subcommands and still
+// allow other functions to directly access each target individually.
+var (
+	installConfigTarget = target{
+		name: "Install Config",
+		command: &cobra.Command{
+			Use:   "install-config",
+			Short: "Generates the Install Config asset",
+			Long:  "",
```

Comment:
While we're touching these, can we either drop or fill in the `Long` values?  I don't see a point to explicitly setting the zero value.